data/json/flags.json

```diff
@@ -2141,6 +2141,16 @@
     "type": "json_flag",
     "info": "This is meant for exosuit foot plating."
   },
+  {
```

Comment:
Am I missing something here or are those just used to group the robots into the pocket whitelist? If so... why bother with flags and not just manually define a whitelist for those pockets with specific item IDs?

Reply:
Np. Mostly because I intend to end up with 15 sellable robots or so, and dont want to copy paste a list of that size around.